src/ainft.ts

```diff
@@ -141,12 +148,21 @@ export default class AinftJs {
     this.setSigner(signer);
   }
 
+  async open() {
+    const privateKey = this.ain.wallet.defaultAccount?.private_key;
+    await (privateKey ? this.ainize.login(privateKey) : this.ainize.loginWithSigner());
```

Comment:
minor) how about splitting it to multi lines? in such a case there is some error, it'd easier to identify which call caused that from the call stack?

Reply:
done!